include/ccl_emu17.h

```diff
@@ -4,6 +4,7 @@
 #define A_MIN_EMU 1./3.
 #define K_MAX_EMU 5.0
 #define K_MIN_EMU 1.0000000474974513E-003
+#define NK_EMU 351
```

Comment:
In ccl_emu.c, you should use `NK_EMU` now in place of `nmode` (which is currently set to 351 near the top).